docs/guides/tcptracer.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'The "tcptracer" gadget'
+weight: 10
+---
+
+The `tcptracer` gadget is used to monitor tcp connections.
+
+## How to use it?
+
+First, we need to create one pod with limited amount of memory:
+
+```bash
+$ $ kubectl run busybox --image busybox:latest sleep inf
+pod/busybox created
+```
+
+You can now use the gadget, but output will be empty:
+
+```bash
+$ kubectl gadget tcptracer
+NODE             NAMESPACE        POD              CONTAINER        KPID   KCOMM            PAGES  TPID             TCOMM
+```
+
+Indeed, it is waiting for TCP connection to be established in the `default` namespace (you can use `-A` to monitor all namespaces and then be sure to not miss any event).
+So, in *an other terminal*, `exec` a container and run this `wget`:
+
+```bash
+$ kubectl exec -ti busybox -- wget https://www.microsoft.com
+Connecting to www.microsoft.com (95.101.225.221:443)
+wget: note: TLS certificate validation not implemented
+wget: server returned error: HTTP/1.1 403 Forbidden
+command terminated with exit code 1
+```
+
+Go back to *the first terminal* and see:
+
+```bash
+NODE             NAMESPACE        POD              CONTAINER        T PID    COMM             IP  SADDR            DADDR            SPORT   DPORT
+minikube         <>               <>               <>               C 19981  wget             4   172.17.0.3       95.101.225.221   34434   443
```

Comment:
What about also printing the line when the connection is closed?

Reply:
I do not understand your comment.
I am only getting one line when I run `wget`.

Reply:
I think you get one line because the connection does not success. Now that you are using `kinvolk.io` you should get another entry for the `close()`.

Reply:
I tested with `kinvolk.io` but I am still getting only one line.
Maybe it is because I use `podinformer`.

Reply:
It's not related with podinformer at all. It's something with the `wget` implementation of busybox. I tried other images and two lines are printed.

Reply:
Thank you for your test, let's stick with this output.

Reply:
IMO, the container image should be changed to be able to show the `close()` entry in the output which will contain the Kubernetes metadata. I think it would improve the guide because with the current example you are hitting https://github.com/kinvolk/inspektor-gadget/issues/550 and no Kubernetes metadata is shown at all. WDYT?

Reply:
I do not really see the point.
The important thing is that user gets the same output when they test than that of the guide.
So, as long as they test with `busybox` they will only get one line for `connect()`.

Regarding #550, we can either wait for it to be solved and I will update this guide or we can update the guide in #550.